### ISSUE TITLE
getter-return: does not check `Object.create` and `Object.defineProperties`

### ISSUE THREAD

**Issue by hirasawayuki** [denoland/deno_lint#1088]
### Lint Name

getter-return
### Code Snippet

```ts
Object.create(Object.prototype, {
  foo: {
    writable: true,
    configurable: true,
    value: 'hello'
  },
  bar: {
    configurable: false,
    get: function() {},
    set: function(value) {
      console.log('Setting `o.bar` to', value);
    }
  }
});
```

```ts
Object.defineProperties(obj, {
  'property1': {
    value: true,
    writable: true,
    get: () => { return true }
  },
  'property2': {
    value: 'Hello',
    writable: false,
    get: () => {} // ← no warnings
  }
});
```

### Expected Result
warnings on getters which have no return values.

### Actual Result
no warning on `Object.create` and `Object.defineProperties` usage.

### Version
deno 1.25.0 (release, aarch64-apple-darwin)
v8 10.6.194.5
typescript 4.7.4
